gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -604,6 +604,32 @@ def self.visibility_method_name(mod, name)
       mod.method(name)
     end
   end
+
+  def self.unload_const(mod)
+    unload_const_sigs(mod)
+    unload_const_sigs(mod.singleton_class)
+    mod.constants(false).each do |const_name|
+      # The inner module could be already unloaded. By other
+      # parent module
```

Comment:
This comment does not match the code now.


```suggestion
      # Because this code is run when a code reloader is about to unload a constant, triggering an inner
      # module autoload will probably confuse the reloader.
```